frontend/packages/ceph-storage-plugin/src/components/bucket-class/wizard-pages/namespace-store-pages/multi-namespace-store.tsx

```diff
@@ -58,7 +58,7 @@ export const MultiNamespaceStorePage: React.FC<MultiNamespaceStoreProps> = React
             </Title>
             <p className="nb-create-bc-step-page-form__element--light-text">
               {t(
-                'ceph-storage-plugin~Select list of namespace stores, defines the read targets of the namespace bucket',
+                'ceph-storage-plugin~Select a list of NamespaceStore which defines the read targets of the namespaced bucket',
```

Comment:
Select a list of Namespace stores that defines the read targets of the namespace bucket.